crates/red_knot_module_resolver/src/typeshed.rs

```diff
@@ -16,7 +18,7 @@ mod tests {
 
         let path_to_functools = Path::new("stdlib").join("functools.pyi");
         let mut functools_module_stub = typeshed_zip_archive
-            .by_name(path_to_functools.to_str().unwrap())
+            .by_name(&path_to_functools.to_slash().unwrap())
```

Comment:
Rather than adding the dependency, I recommend to construct the path as a normal string: `"stdlib/functools.pyi"`, seems simpler and removes the need for a dependency

Reply:
Good point, this isn't needed here